docs/src/token.md

```diff
@@ -2,9 +2,76 @@
 title: Token Program
 ---
 
-An ERC20-like Token program on the Solana blockchain.
+A Fungible Token program on the Solana blockchain.
 
-The project comprises:
+This program provides an interface and implementation that third parties can use to create and trade their tokens.
 
-* The Rust on-chain program
-* A JavaScript library to interact with the on-chain program
+## Background
+
+Solana's programming model and the definitions of the Solana terms used in this document are available at:
+- https://docs.solana.com/apps
+- https://docs.solana.com/terminology
+
+## Source
+
+The Token Program's source is available on [github](https://github.com/solana-labs/solana-program-library)
+
+## Interface
+
+The on-chain Token Program is written in Rust and available on crates.io as [spl-token](https://docs.rs/spl-token/1.0.2/spl_token/) documents the program's [instruction interface documentation](https://docs.rs/spl-token/1.0.2/spl_token/instruction/enum.TokenInstruction.html)
+
+Auto-generated C bindings are also available for the on-chain Token Program and available [here](https://github.com/solana-labs/solana-program-library/blob/master/token/inc/token.h)
+
+[Client side bindings](https://github.com/solana-labs/solana-program-library/blob/master/token/js/client/token.js) are available in Javascript to load a Token Program and issue instructions
+
+[Javascript bindings](https://github.com/solana-labs/solana-program-library/blob/master/token/js/client/token.js) are available that support loading the Token Program on to a chain and issuing instructions.
+
+## Operational overview
+
+### Creating a new token type
+
+A new token type can be created by initializing a new Mint with the `InitializeMint` instruction.  The Mint is used to create or "Mint" new tokens, and these tokens are stored in Accounts.  A Mint is associated with each Account, which means that the total supply of a particular token type is equal to the balances of all the associated Accounts.
+
+A Mint can either be configured with a fixed-supply or not.  The total supply of a fixed-supply Mint is determined during initialization and deposited into a provided destination account.  A non-fixed-supply Mint also has an owner associated with it who has the authority to create new tokens in the future with the `MintTo` instruction.
```

Comment:
```suggestion
A Mint can either be configured with a fixed or non-fixed supply.  The total supply of a fixed-supply Mint is determined during initialization and deposited into a provided destination account.  A non-fixed-supply Mint has an owner associated with it who has the authority to create new tokens in the future with the `MintTo` instruction.
```

Reply:
I think it's worth mentioning burn instructions here because AFAICT you can still burn "fixed-supply" mint tokens which would decrease the supply